src/node/server/serverPluginCss.ts

```diff
@@ -99,6 +101,7 @@ export const cssPlugin: ServerPlugin = ({
             ...(expectsModule
               ? [
                   require('postcss-modules')({
+                    generateScopedName: `[local]_${id}`,
                     getJSON(_: string, json: Record<string, string>) {
```

Comment:
For module css can generate scope name.So we don't treat it to js update.it can be updated with same as normal css.